Resources/Prototypes/Datasets/ion_storm.yml

```
@@ -362,40 +336,46 @@
   - ЧЛЕНЫ ЭКИПАЖА
   - КИБОРГИ
   - ДЕТЕКТИВЫ
-  # - DRONES / uncomment if/when drones get reenabled
-  # - GENETICISTS
+  # - ГЕНЕТИКИ
   - ГЛАВЫ ПЕРСОНАЛА
   - ГЛАВЫ СЛУЖБЫ БЕЗОПАСНОСТИ
   - ГЛАВЫ ОТДЕЛОВ
   - УБОРЩИКИ
-  - АГЕНТЫ ВНУТРЕННЫХ ДЕЛ
+  - АДВОКАТЫ
```

бля, на корваксе же этот гандон
```suggestion
  - АГЕНТЫ ВНУТРЕННИХ ДЕЛ
```